.github/workflows/renovate.yml

```
@@ -0,0 +1,30 @@
+name: Renovate
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every night
+    - cron: "0 0 * * *"
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        # Adding `fetch-depth: 0` makes sure tags are also fetched. We need
+        # the tags so `git describe` returns a valid version.
+        # see https://github.com/actions/checkout/issues/701 for extra info about this option
+        with: { fetch-depth: 0 }
+
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v43.0.5
```

```suggestion
        uses: renovatebot/github-action@85b17ebd5abf43d1c34c01bd4c8dbb8d45bbc2c7 # v43.0.7
```

We should always use pinned GHA to reduce the possibility of a security incident.

---

Yup.. forgot about that